EIPIP Meeting 25 Notes.md

```diff
@@ -0,0 +1,318 @@
+# EIPIP Meeting 25 Notes
+## Meeting Date/Time: Wednesday 27 January at 15:00 UTC
+### Meeting Duration: 1 hour
+### [GitHub Agenda Page](https://github.com/ethereum-cat-herders/EIPIP/issues/48)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=zz1IhYKztuc)
+### Moderator: Pooja Ranjan
+### Notes: Shane Lightowler
+
+----
+## DECISION ITEMS
+
+**DECISION**: The new Cat Herders website should focus on being an index/intro rather than an in-depth content repository. It should point to canonical sources elsewhere (eg Eth1.0 specs repo, Ethereum.org) to avoid duplication [16:03](https://youtu.be/zz1IhYKztuc?t=963)
```

Comment:
```suggestion
**DECISION 25.1**: The new Cat Herders website should focus on being an index/intro rather than an in-depth content repository. It should point to canonical sources elsewhere (eg Eth1.0 specs repo, Ethereum.org) to avoid duplication [16:03](https://youtu.be/zz1IhYKztuc?t=963)
```